index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./style.css">
+  <title>Document</title>
+</head>
+
+
+<body>
+  <div class="main">
+    <div class="top-bar">
+      <div class="nav-">
+        <nav id="navbar">
+          <ul id="list">
+            <li><a href="#about">About </a></li>
+            <li><a href="#projects">Work</a></li>
+            <li><a href="#contacts">Contact</a></li>
+          </ul>
+        </nav>
+      </div>
+    </div>
+
+    <section id="welcome-section">
+      <h1>Hey I am Mimic</h1>
+      <p>a web developer</p>
+    </section>
+
+    <section id="projects">
+      <h1 class="project-tile">These are some of my projects</h1>
+      <div class="project-list">
+        <div class="grid">
+          <a href="#">
+            <img src="img/1.jpeg" alt="image"/>
+            <p>Tribute</p>
+          </a>
+        </div>
+        <div class="grid">
+          <a href="#">
+            <img src="img/2.png" alt="" />

```

Comment:
- [ ] Kindly remember to put a meaningful message into your  ` alt ` attribute to describe the added image. This will not only increase your work look more professional but even will increase the user experience for SEO (search engine optimization) :100: . For more information, you can read [this article](https://www.innovationvisual.com/knowledge/why-image-alt-text-is-important-for-seo).